test/unit/test_http_retries.py

```diff
@@ -40,6 +41,7 @@ def test_create_detector_attempts_retries(gl: Groundlight):
     )
 
 
+@flaky(max_runs=4, min_passes=1)
 def test_get_or_create_detector_attempts_retries(gl: Groundlight):
     run_test(
         mocked_call="urllib3.PoolManager.request",
```

Comment:
The issue with this test is that it is mocking the wrong thing, it should be mocking `requests.request`: https://github.com/groundlight/python-sdk/blob/main/src/groundlight/internalapi.py#L235

```suggestion
        mocked_call="requests.request",
```